lib/knapsack_pro/config/ci/base.rb

```diff
@@ -32,6 +32,17 @@ def detected
         def fixed_queue_split
           true
         end
+
+        def ci_provider
+          return 'AWS CodeBuild' if ENV.key?('CODEBUILD_BUILD_ARN')
+          return 'Azure Pipelines' if ENV.key?('SYSTEM_TEAMFOUNDATIONCOLLECTIONURI')
+          return 'Bitbucket Pipelines' if ENV.key?('BITBUCKET_COMMIT')
+          return 'Google Cloud Build' if ENV.key?('BUILDER_OUTPUT')
+          return 'Jenkins' if ENV.key?('JENKINS_URL')
+          return 'TeamCity' if ENV.key?('TEAMCITY_VERSION')
+
+          "Other"
```

Comment:
```suggestion
          "Other" if ENV.key?('CI')
```

Maybe we should do something like that. 

Thanks to that if you are running knapsack inside of the CI then it's `Other` (not perfect solution because we can't probably assume each CI has `CI=true` value set. But most do it so maybe that's the good enough assumption).
If you run it in the development then we won't override the existing detected CI on the API side by the `Other` value.

Reply:
https://github.com/KnapsackPro/knapsack_pro-ruby/pull/216/commits/3c3881f885641c0424fbfced38ca2186cac422d3